app/controllers/admin/searchgov_domains_controller.rb

```diff
@@ -1,10 +1,13 @@
 class Admin::SearchgovDomainsController < Admin::AdminController
   active_scaffold :searchgov_domain do |config|
     config.label = 'Search.gov Domains'
-    config.actions = %i[create list search export nested]
+    config.actions = %i[create list search export nested show]
```

Comment:
You can skip adding the `show` link; I was hoping to include the sitemaps in that view, but now that we're displaying sitemaps separately, the `show` view doesn't show anything that's not already displayed in the list view.